src/shared/containers/Refile/RefileErrorsForm.jsx

```diff
@@ -0,0 +1,236 @@
+  * @param {object} state - the current state object
+  * @param {boolean} focusOnError - flag utilized to execute focusOnField() if true (default: false)
+  */
+  validateField(fieldName, state, focusOnError = false) {
+    switch (fieldName) {
+      case 'startDate':
+        // TODO: implement validations, make sure 1) there's value; 2) the value is in the valid format
+        break;
+      case 'endDate':
+        // TODO: implement validations, make sure 1) there's value; 2) the value is in the valid format
+        // if (!isBarcodeValid(state.formFields[fieldName])) {
+        //   this.setFieldError(fieldName, state, 'The barcode field must be 14 numerical characters');
+        //   if (focusOnError === true) {
+        //     this.focusOnField(fieldName);
+        //   }
+        // } else {
+        //   this.removeFieldError(fieldName, state);
+        // }
+        break;
+      default:
+        break;
+    }
+  }
+
+  /**
+  * @desc Handles sending the form field payload from the state to the proper API endpoint. All
+  * fields are validated prior to executing the ajax call. Updates the form state booleans and result
+  * based on successful or error responses
+  * @param {object} event - contains the current event context of the field
+  */
+  handleFormSubmit(event) {
+    event.preventDefault();
+
+    // TODO: execute validations here
+    // Iterate through patron fields and ensure all fields are valid
+    // forIn(this.state.formFields, (value, key) => {
+    //   this.validateField(key, this.state, true);
+    // });
+
+    if (isEmpty(this.state.fieldErrors)) {
+      const {
+        type,
+        formFields: {
+          startDate,
+          endDate,
+        }
+      } = this.state;
+
+      // Update the Parent Container Loading State
+      this.props.setApplicationLoadingState(true);
+
+      return axios.post(
+        '/get-refile-errors',
+        {
+          startDate,
+          endDate,
+        },
+        {
+          headers: { 'csrf-token': this.state.csrfToken }
+        }
+      ).then(response => {
+        console.log('Form Successful Response: ', response);
+        this.props.setApplicationLoadingState(false);
+        this.setState({...this.baseState, formResult: { processed: true } });
+      }).catch(error => {
+        console.log('Form Error Response: ', error);
+        this.props.setApplicationLoadingState(false);
+        this.setState({...this.state, formResult: { processed: false, response: error } });
+      });
+    }
+  }
+
+  /**
+  * @desc Handles returning the correct DOM for the Refile Errors form
+  */
+  renderRefileErrorsFrom() {
+    return(
+      <form onSubmit={this.handleFormSubmit}>
+        <FormField
+          className="nypl-text-field"
+          id="startDate"
+          type="text"
+          label="StartDate"
+          fieldName="startDate"
+          instructionText="Please enter a 14 digit barcode"
```

Comment:
Guess this is the wrong instruction! :-D

Reply:
Yeah it is!! And you can see I have updated them in the later PR.

Reply:
Cool!